internal/schemavalidator/at_least_one_of_test.go

```diff
@@ -232,7 +249,64 @@ func TestAtLeastOneOfValidatorValidate(t *testing.T) {
 			in: path.Expressions{
 				path.MatchRoot("fooz"),
 			},
-			expErrors: 2,
+			expected: &schemavalidator.AtLeastOneOfValidatorResponse{
+				Diagnostics: diag.Diagnostics{
+					diag.NewErrorDiagnostic(
+						"Invalid Path Expression for Schema",
+						"The Terraform Provider unexpectedly provided a path expression that does not match the current schema. "+
+							"This can happen if the path expression does not correctly follow the schema in structure or types. "+
+							"Please report this to the provider developers.\n\nPath Expression: fooz",
+					),
+					diag.WithPath(
+						path.Root("bar"),
+						diag.NewErrorDiagnostic(
+							"Invalid Attribute Combination",
+							"At least one attribute out of [fooz,bar] must be specified",
+						),
+					),
+				},
+			},
+		},
+		"both-attrs-in-diag-msg": {
+			req: schemavalidator.AtLeastOneOfValidatorRequest{
+				ConfigValue:    types.NumberNull(),
+				Path:           path.Root("bar"),
+				PathExpression: path.MatchRoot("bar"),
+				Config: tfsdk.Config{
+					Schema: schema.Schema{
+						Attributes: map[string]schema.Attribute{
+							"foo": schema.Int64Attribute{},
+							"bar": schema.Int64Attribute{},
+						},
+					},
+					Raw: tftypes.NewValue(
+						tftypes.Object{
+							AttributeTypes: map[string]tftypes.Type{
+								"foo": tftypes.Number,
+								"bar": tftypes.Number,
+							},
+						},
+						map[string]tftypes.Value{
+							"foo": tftypes.NewValue(tftypes.Number, nil),
+							"bar": tftypes.NewValue(tftypes.Number, nil),
+						},
+					),
+				},
+			},
+			in: path.Expressions{
+				path.MatchRoot("foo"),
+			},
+			expected: &schemavalidator.AtLeastOneOfValidatorResponse{
+				Diagnostics: diag.Diagnostics{
+					diag.WithPath(
+						path.Root("bar"),
+						diag.NewErrorDiagnostic(
+							"Invalid Attribute Combination",
+							"At least one attribute out of [foo,bar] must be specified",
+						),
+					),
+				},
+			},
 		},
```

Comment:
```suggestion
```

I believe that this test is no longer strictly necessary, as both `matches-no-attribute-in-schema`, and `error_none-set` tests are verifying that all of the relevant attribute names appear in the error message.

Reply:
Done, thanks!